src/calculateRentalCost.js

```diff
@@ -4,7 +4,31 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentDayCost = 40;
+  const longTermOfRent = 7;
+  const longTermDiscount = 50;
+  const shortTermOfRent = 3;
+  const shortTermDiscount = 20;
+
+  const cost = days * rentDayCost;
+
+  if(days >= longTermOfRent) {
+    return cost - longTermDiscount;
+  }
+
+  if(days >= shortTermOfRent ) {
+    return cost - shortTermDiscount;
+  }
```

Comment:
The logic here is redundant. You have two sets of conditions for applying discounts based on the number of days (`days >= longTermOfRent` and `days > 6`, `days >= shortTermOfRent` and `days > 2`). This can lead to confusion and incorrect discount application. Consider removing the duplicate conditions and ensuring that only one set of conditions is used to apply the correct discount.